File: fabric/src/main/java/net/impleri/mobskills/fabric/MobSkillsFabric.java
package net.impleri.mobskills.fabric;

import net.fabricmc.api.ModInitializer;
import net.impleri.mobskills.MobSkills;

public class MobSkillsFabric implements ModInitializer {
    @Override
    public void onInitialize() {
        MobSkills.init();
    }
}

File: fabric/src/main/kotlin/net/impleri/mobskills/client/fabric/MobSkillsFabric.kt
package net.impleri.mobskills.client.fabric

import net.fabricmc.api.ModInitializer
import net.impleri.mobskills.MobSkills

class MobSkillsFabric : ModInitializer {
  override fun onInitialize() {
    MobSkills.init()
  }
}
